Simplenote/src/androidTest/java/com/automattic/simplenote/models/NoteIndexerTest.java

```diff
@@ -21,33 +21,80 @@
 @RunWith(AndroidJUnit4.class)
 @SmallTest
 public class NoteIndexerTest {
+    private static final String DATE_STRING = "2013-09-12T20:47:03.000Z";
+    private static final double DATE_DOUBLE = 1379018823037.0;
+    private static final long DATE_LONG = 1379018823000L;
+    private static final long DATE_NUMBER = 1379018823;
 
+    private Note mNote;
     private NoteIndexer mIndexer;
 
     @Before
-    public void setUp() {
+    public void setUp() throws JSONException {
+        JSONObject properties = new JSONObject();
+        properties.put(Note.CREATION_DATE_PROPERTY, DATE_DOUBLE);
+        properties.put(Note.MODIFICATION_DATE_PROPERTY, DATE_DOUBLE);
+        properties.put(Note.PUBLISH_URL_PROPERTY, "test.url.property");
+        mNote = new Note("test", properties);
         mIndexer = new NoteIndexer();
     }
 
     @Test
     public void testIndexMillisecondDates() throws JSONException {
```

Comment:
```suggestion
    public void testIndexMillisecondDates() {
```

We can remove `throws JSONException` since we're no longer putting properties in a JSON object.